roles/matrix-bot-honoroit/templates/systemd/matrix-bot-honoroit.service.j2

```diff
@@ -0,0 +1,51 @@
+#jinja2: lstrip_blocks: "True"
+[Unit]
+Description=Matrix helpdesk bot
+{% for service in matrix_bot_honoroit_systemd_required_services_list %}
+Requires={{ service }}
+After={{ service }}
+{% endfor %}
+{% for service in matrix_bot_honoroit_systemd_wanted_services_list %}
+Wants={{ service }}
+{% endfor %}
+DefaultDependencies=no
+
+[Service]
+Type=simple
+Environment="HOME={{ matrix_systemd_unit_home_path }}"
+ExecStartPre=-{{ matrix_host_command_sh }} -c '{{ matrix_host_command_docker }} kill matrix-bot-honoroit 2>/dev/null'
+ExecStartPre=-{{ matrix_host_command_sh }} -c '{{ matrix_host_command_docker }} rm matrix-bot-honoroit 2>/dev/null'
+
+ExecStart={{ matrix_host_command_docker }} run --rm --name matrix-bot-honoroit \
+			--log-driver=none \
+			--user={{ matrix_user_uid }}:{{ matrix_user_gid }} \
+			--cap-drop=ALL \
+			--read-only \
+			--network={{ matrix_docker_network }} \
+			-e 'HONOROIT_LOGIN={{ matrix_bot_honoroit_login }}' \
+			-e 'HONOROIT_PASSWORD={{ matrix_bot_honoroit_password }}' \
+			-e 'HONOROIT_HOMESERVER={{ matrix_bot_honoroit_homeserver }}' \
+			-e 'HONOROIT_ROOMID={{ matrix_bot_honoroit_roomid }}' \
+			-e 'HONOROIT_DB_DSN={{ matrix_bot_honoroit_database_connection_string }}' \
+			-e 'HONOROIT_DB_DIALECT={{ matrix_bot_honoroit_database_dialect }}' \
+			-e 'HONOROIT_PREFIX={{ matrix_bot_honoroit_prefix }}' \
+			-e 'HONOROIT_SENTRY={{ matrix_bot_honoroit_sentry }}' \
+			-e 'HONOROIT_LOGLEVEL={{ matrix_bot_honoroit_loglevel }}' \
+			-e 'HONOROIT_TEXT_GREETINGS={{ matrix_bot_honoroit_text_greetings }}' \
+			-e 'HONOROIT_TEXT_ERROR={{ matrix_bot_honoroit_text_error }}' \
+			-e 'HONOROIT_TEXT_EMPTYROOM={{ matrix_bot_honoroit_text_emptyroom }}' \
+			-e 'HONOROIT_TEXT_DONE={{ matrix_bot_honoroit_text_done }}' \
```

Comment:
It will be better if all of these environment variables are moved to a file. They can then be referenced here using a single line: `--env-file=PATH`.

See how the Jitsi role does it for one of its components: https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/7dfb68b7f82fb09edbcdcad2a98875be2c3dda42/roles/matrix-jitsi/templates/web/matrix-jitsi-web.service.j2#L20

The file template itself looks like this (fairly large in this case): https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/7dfb68b7f82fb09edbcdcad2a98875be2c3dda42/roles/matrix-jitsi/templates/web/env.j2

It's also **user-extensible using a variable**. See: https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/7dfb68b7f82fb09edbcdcad2a98875be2c3dda42/roles/matrix-jitsi/templates/web/env.j2#L109-L110

Thanks to this user-extensibility, you don't need to expose each and every future environment variable supported by Honoroit as a playbook variable. Obscure settings can be left out. Users who need them can use `matrix_bot_honoroit_environment_variables_extension`.

That template file is processed and put on the server using a `template` module task like this: https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/7dfb68b7f82fb09edbcdcad2a98875be2c3dda42/roles/matrix-jitsi/tasks/setup_jitsi_web.yml#L29-L34

-------

Benefits of going for an environment variable file:

- visually cleaner systemd `.service` file
- changing environment variables does **not** necessitate `systemctl daemon-reload` when they are specified in a file
- secrets specified in environment variables do not leak (if passed using `-e`, any user on the server that uses `ps` can see your full `docker run ..` command and all secrets inside (`HONOROIT_PASSWORD` and `HONOROIT_DB_DSN`))